root/utility/tableColumns.js

```diff
@@ -98,13 +98,13 @@ export function defineArtistCreditColumn<D>(
     Cell: ({row: {original}}) => {
       const artistCredit = props.getArtistCredit(original);
       return (artistCredit
-        ? props.showExpandedArtistCredits
+        ? props.showExpandedArtistCredits /*:: === true */
```

Comment:
Should we destructure these with default values instead?
```js
const {
  showExpandedArtistCredits = false,
  sortable = false,
} = props;
```
That seems a bit clearer than the flow-includes.